src/fillalgebra.jl

```diff
@@ -1,3 +1,10 @@
+const FillVector{F,A} = Fill{F,1,A}
```

Comment:
I don't see why we need these as `Fill{F,1}` is shorter than `FillVector{F}`

Reply:
because `FillVector` is nicer than   than  `Fill{F,1}  ... where F`  or `Fill{<:Any,1}`

Reply:
Nicer is in the eye of the beholder... but I concede your point